<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-12-24T23:40:14 IST" name="Suite" finished-at="2023-12-24T23:41:12 IST" duration-ms="58733">
    <groups>
    </groups>
    <test started-at="2023-12-24T23:40:14 IST" name="Test" finished-at="2023-12-24T23:41:12 IST" duration-ms="58733">
      <class name="TestCases.Test_ValidLogin">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:TestCases.Test_ValidLogin@3e27ba32]" started-at="2023-12-24T23:40:56 IST" name="launchApplication" finished-at="2023-12-24T23:41:08 IST" duration-ms="11851" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:TestCases.Test_ValidLogin@3e27ba32]" started-at="2023-12-24T23:41:08 IST" name="setUp" finished-at="2023-12-24T23:41:08 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="test_validLogin()[pri:0, instance:TestCases.Test_ValidLogin@3e27ba32]" started-at="2023-12-24T23:41:08 IST" name="test_validLogin" finished-at="2023-12-24T23:41:12 IST" duration-ms="3871" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_validLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestCases.Test_ValidLogin@3e27ba32]" started-at="2023-12-24T23:41:12 IST" name="tearDown" finished-at="2023-12-24T23:41:12 IST" duration-ms="520" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCases.Test_ValidLogin -->
      <class name="TestCases.Test_InvalidLogin">
        <test-method is-config="true" signature="testExecution()[pri:0, instance:TestCases.Test_InvalidLogin@2af004b]" started-at="2023-12-24T23:40:14 IST" name="testExecution" finished-at="2023-12-24T23:40:14 IST" duration-ms="23" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testExecution -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:TestCases.Test_InvalidLogin@2af004b]" started-at="2023-12-24T23:40:14 IST" name="launchApplication" finished-at="2023-12-24T23:40:32 IST" duration-ms="17989" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:TestCases.Test_InvalidLogin@2af004b]" started-at="2023-12-24T23:40:32 IST" name="setUp" finished-at="2023-12-24T23:40:32 IST" duration-ms="57" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="test_InvalidLogin()[pri:0, instance:TestCases.Test_InvalidLogin@2af004b]" started-at="2023-12-24T23:40:32 IST" name="test_InvalidLogin" finished-at="2023-12-24T23:40:55 IST" duration-ms="23814" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[invalid login error message not displayed expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: invalid login error message not displayed expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at TestCases.Test_InvalidLogin.test_InvalidLogin(Test_InvalidLogin.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_InvalidLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestCases.Test_InvalidLogin@2af004b]" started-at="2023-12-24T23:40:56 IST" name="tearDown" finished-at="2023-12-24T23:40:56 IST" duration-ms="544" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCases.Test_InvalidLogin -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
